src/test/java/hw1/operations/AbstractOperationTest.java

```diff
@@ -0,0 +1,34 @@
+package hw1.operations;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeTest;
+
+public abstract class AbstractOperationTest {
+
+    protected Calculator calculator;
+
+    @BeforeTest
+    public void beforeTest() {
+        System.out.println("before test");
+    }
+
+    @BeforeMethod
+    public void setUp() {
+        System.out.println("before method");
+        calculator = new Calculator();
```

Comment:
Do we need a new instance every time before the method? It is more suitable to use one instance for the suite.

Reply:
I didn't manage to implement this with BeforeSuite, only with BeforeClass. As far as I understand, this is because the tests are in different classes. For now, I do not understand how to rewrite code to work with BeforeSuite.